source/ota.c

```diff
@@ -2871,7 +2871,6 @@ static void receiveAndProcessOtaEvent( void )
     }
 }
 
-/* Event Processing loop to run the OTA state machine. */
```

Comment:
Could we add this/or a better description back?

Reply:
I removed this to be consistent with the rest of the file. The rest of the functions that are in ota.h have a description there (that is more in depth than this comment was) and no description in the ota.c file.

Reply:
I would suggest adding comments to the rest of the functions(be it one liners), rather than removing them.